### ISSUE TITLE
unexpected output with array range syntax

### ISSUE THREAD

**Issue by unlessgames** [DanielXMoore/Civet#889]
```coffee
fun 
  a : 0

[0...128].map (i) => console.log i
```

compiles into

```js
fun({
  a: 0,
})((s, e) => {
  let step = e > s ? 1 : -1;
  return Array.from(
    { length: Math.abs(e - s) },
    (_, i) => s + i * step
  );
})(0, 128).map((i) => console.log(i));
```

**Comment by edemaine** [comment]
FWIW, I'm guessing that this is an ASI (Automatic Semicolon Insertion) issue.

**Comment by edemaine** [comment]
Thanks for the report! Should be fixed in 0.6.60.
